pkg/core/native/nonfungible.go

```diff
@@ -247,6 +247,8 @@ func (n *nonfungible) tokensOf(ic *interop.Context, args []stackitem.Item) stack
 	return stackitem.NewInterop(iter)
 }
 
+var _ = (*nonfungible).mint // fix unused warning
```

Comment:
Does this library make any sense at all after native NNS removal?

Reply:
Fixed.